language/de/cli.php

```diff
@@ -67,6 +68,7 @@
 	'CLI_DESCRIPTION_RECALCULATE_EMAIL_HASH'	=> 'Berechnet die Einträge im Feld user_email_hash der Benutzer-Tabelle neu.',
 	'CLI_DESCRIPTION_SET_ATOMIC_CONFIG'			=> 'Legt den Wert für eine Konfigurations-Einstellung fest, wenn die derzeitige Einstellung dem angegebenen Vergleichswert entspricht',
 	'CLI_DESCRIPTION_SET_CONFIG'				=> 'Legt den Wert für eine Konfigurations-Einstellung fest',
+	'CLI_DESCRIPTION_UPDATE_HASH_BCRYPT'		=> 'Aktualisiert veraltete Passwort Hashes und hasht diese mit bcrypt.',
```

Comment:
Ich hab mal Marc gefragt, er hat gesagt, dass diese neue Funktion die alten Hashes nochmal mit Bcrypt hashet (doppelt) und beim nächsten Login dann nur noch alleine mit bcrypt. So liegen die Hashes nur noch in Bcrypt vor und müssen nicht geknackt werden und man muss nicht warten, bis die User sich mal wieder anmelden, was viele eben nicht tun.